backend/tools/event_sender/src/main.rs

```diff
@@ -16,10 +15,18 @@ use rand::{thread_rng, Rng};
 #[derive(Parser)]
 #[clap(about = "Generates random events and sends them over ampq")]
 struct Cli {
-    /// Total amount of events to be sent (note: multiplied with the `burst` option)
-    #[clap(default_value = "1", short, long)]
+    // NOTE: The number of events along with the total duration, min lacency and latency max is to be used for stress testing.
+    // When setting these values, keep in mind that the generator process will stop at whichever value is reached first (between the count and total duration).
+    // The defaut settings on these variables represent roughly 30,000 events sent per hour.
+
+    // Total amount of events to be sent (note: multiplied with the `burst` option)
+    #[clap(default_value = "30000", short, long)]
     count: usize,
 
+    // Total duration to run the event generator
```

Comment:
All these comments need to be tripple slash, otherwise it doesn't show up in the --help command:
![image](https://user-images.githubusercontent.com/57626200/201388888-3619f105-0fe9-4a20-837d-d21e5dda4d6d.png)

Reply:
ping me when ready, looks good to merge.